tsconfig.json

```diff
@@ -39,5 +39,9 @@
     "strictInjectionParameters": true,
     "strictInputAccessModifiers": true,
     "strictTemplates": true
-  }
+  },
+  "include": [
+    "./node_modules/@piwikpro/tracking-base-library/dist",
+    "./projects"
+  ],
```

Comment:
Question: why do we need to include that?

Reply:
For the typedocs reason. It wouldn't build otherwise

Reply:
Builds fine for me, maybe it is not needed anymore after https://github.com/PiwikPRO/ngx-piwik-pro/pull/32

Reply:
done